src/Engine/ProtoCore/Reflection/GraphicDataProvider.cs

```diff
@@ -141,8 +141,17 @@ internal List<IGraphicItem> GetGraphicItems(DSASM.StackValue svData, RuntimeCore
             return null;
         }
 
+        //Store marshaller for repeated calls to GetCLRObject.  Used for short-circuit of re-allocation of marshaller / interpreter object.
+        private static Interpreter interpreter;
+        private static ProtoFFI.FFIObjectMarshaler marshaler;
+        
         internal object GetCLRObject(StackValue svData, RuntimeCore runtimeCore)
         {
+            if (marshaler != null)
```

Comment:
Do we also need to do a null check for the interpreter here?

Reply:
I don't think so....  The marshaler and interpreter are assigned at the same time.  But maybe good just to be sure.